lib/utf8.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+module.exports.decode = function (input) {
+  if (Buffer.isBuffer(input) === false) {
```

Comment:
I see. I think the correct fix is for line 121 to be `utf8.decode(utf8.encode(output))`